libsecp256k1.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "libsecp256k1",
+    "config-opts": [ "--disable-static" ],
```

Comment:
```suggestion
    "config-opts": [
        "--enable-module-recovery",
        "--disable-static"
    ],
```
Is needed for electrum 4.0.0.

Reply:
ok, I'll commit newer commit and remove `--disable-static` in a hour

Reply:
Disable static is fine, add ` "--enable-module-recovery",`.

Reply:
Yes, sorry.

Reply:
added suggestion in 3bdca52